tests/unit/Statement/StatementListTest.php

```diff
@@ -702,4 +703,48 @@ public function testGivenNoTraversable_setStatementsThrowsException() {
 		$statements->setStatements( $statement1 );
 	}
 
+	public function testGivenNoStatement_setStatementsThrowsException() {
+		$statement1 = $this->getStatement( 1, 'guid1' );
+		$statements = new StatementList();
+
+		$this->setExpectedException( 'InvalidArgumentException' );
+		$statements->setStatements( array( $statement1, false ) );
+	}
+
+	public function testGivenNoTraversable_setStatementsDoesNotEditList() {
+		$statement1 = $this->getStatement( 1, 'guid1' );
```

Comment:
One statement is enough, indeed. However, it does not hurt to have more of them. Is this an issue?